stock_split_picking/__manifest__.py

```diff
@@ -1,37 +1,25 @@
 # -*- coding: utf-8 -*-
-#
-#
-#    Author: Nicolas Bessi, Guewen Baconnier, Yannick Vaucher
-#    Copyright 2013-2015 Camptocamp SA
-#
-#    This program is free software: you can redistribute it and/or modify
-#    it under the terms of the GNU Affero General Public License as
-#    published by the Free Software Foundation, either version 3 of the
-#    License, or (at your option) any later version.
-#
-#    This program is distributed in the hope that it will be useful,
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU Affero General Public License for more details.
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-#
-{'name': 'Split picking',
- 'summary': 'Split a picking in two unconfirmed pickings',
- 'version': '8.0.1.0.0',
- 'author': "Camptocamp,Odoo Community Association (OCA)",
- 'maintainer': 'Camptocamp',
- 'license': 'AGPL-3',
- 'category': 'Warehouse Management',
- 'complexity': "normal",  # easy, normal, expert
- 'depends': ['stock'],
- 'website': 'http://www.camptocamp.com',
- 'data': ['view/stock_partial_picking.xml'],
- 'demo': [],
- 'test': ['test/test_picking_split.yml',
-          'test/test_assigned_picking_split.yml'],
- 'installable': False,
- 'auto_install': False,
- }
+# Copyright 2013-2015 Camptocamp SA - Nicolas Bessi
+# Copyright 2013-2015 Camptocamp SA - Guewen Baconnier
+# Copyright 2013-2015 Camptocamp SA - Yannick Vaucher
+# Copyright 2017 Tecnativa - Vicent Cubells
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    'name': 'Split picking',
+    'summary': 'Split a picking in two unconfirmed pickings',
+    'version': '10.0.1.0.0',
+    'category': 'Warehouse Management',
```

Comment:
@pedrobaeza  your are right in Odoo 10 the name of the category is Inventory. https://github.com/odoo/odoo/blob/10.0/odoo/addons/base/module/module_data.xml#L39 @cubells Can you take into account the comments from @pedrobaeza?